src/Social/UsersLists/DataTransferObject/UserList.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Social\UsersLists\DataTransferObject;
+
+use Spatie\LaravelData\Data;
+
+class UserList extends Data
+{
+    public function __construct(
+        public int $apps_id,
```

Comment:
@FredPeal i would follow the same pattern as other dto where we pass app, company and user not just the ids